### ISSUE TITLE
Only one display available

### ISSUE THREAD

**Issue by Milliw** [galister/wlx-overlay-s#80]
Hey!

Thx for your software. Did use the old (non s version) a while ago without any flaws.
This one however only asks me once for sharing a screen and this does work, but the second screen is not shown on the panel and is not asked for sharing. I can do it with mirror but thats not great.

What did I do wrong?
I started wlx-overlay in terminal and there was no "order" visible which screen to share first. Did I misunderstood the installation instruction?

**Comment by galister** [comment]
hi, try clear out the pipewire tokens:
`rm ~/.config/wlxoverlay/conf.d/pw_tokens.yaml `

and then restart the overlay

**Comment by Milliw** [comment]
Sorry, did not mention it. I did this already a couple times. Also deleted the whole folder and started over. I get only asked one time.

If I start in terminal, it says found two displays and using one with the key shown. No error AFAIK only one about openxr not available iirc.

Running under Nobara 41 Gnome X11 due to DRM release not available in GNOME 46.
-- Von /e/OS E-Mail gesendet.

Am 23. September 2024 22:13:15 UTC schrieb galister ***@***.***>:

**Comment by galister** [comment]
mind posting your logs from `/tmp/wlx.log`?

**Comment by Milliw** [comment]
When I was looking there yesterday, I did not see any log existing. I'm not at PC ATM. I can look at the evening.
-- Von /e/OS E-Mail gesendet.

Am 24. September 2024 09:45:40 UTC schrieb galister ***@***.***>:

**Comment by Milliw** [comment]
So now I see the log file. Not sure why I did not before. But the content is the same as when I start it in terminal. SteamVR also seems to ignore the autostart though it is set in the vrappconfig file.

Did it over again but still being asked one time for sharing a scree....

Oh wait - forget it. It was my mistake. I did not know, that I can select multiple displays at once to share when the popup comes. So sorry! :D

Only thing that seems to bug - but within SteamVR - I guess is the autostart not happening.

**Comment by galister** [comment]
As far as I know, token saving doesn't work on Gnome, so it'll pop up on every startup, right?

**Comment by Milliw** [comment]
> As far as I know, token saving doesn't work on Gnome, so it'll pop up on every startup, right?

Actually this does work. I have restarted it a couple of times and it remains. No popup again and all displays available.
I added the overlay appimage as a steam game as a workaround for not autostarting. 

My desktop session "crashed" a couple times or at least the system is telling me something bad happened and I need to shut down my user session. I had the beta of SteamVR installed and reverted that in hopes of fixing that. Though I can imagine this could be because of the display sharing. Idk...
